.vscode/mcp.json

```diff
@@ -1,33 +1,33 @@
 {
-  "servers": {
-    "expenses-mcp": {
-      "type": "stdio",
-      "command": "uv",
-      "cwd": "${workspaceFolder}",
-      "args": [
-        "run",
-        "servers/basic_mcp_stdio.py"
-      ]
-    },
-    "expenses-mcp-http": {
-      "type": "http",
-      "url": "http://localhost:8000/mcp"
-    },
-    "expenses-mcp-debug": {
-      "type": "stdio",
-      "command": "uv",
-      "cwd": "${workspaceFolder}",
-      "args": [
-        "run",
-        "--",
-        "python",
-        "-m",
-        "debugpy",
-        "--listen",
-        "0.0.0.0:5678",
-        "servers/basic_mcp_stdio.py"
-      ]
-    }
-  },
-  "inputs": []
-}
+	"servers": {
```

Comment:
What's the whitespace diff from? Pre-commit? (I did add precommit btw if you want to install it)